### DISCUSSION TITLE
Checkbox on the PR to force pending versions

### DISCUSSION THREAD

**Discussion by taraspos** [renovatebot/renovate#30233] [Suggest an Idea]
### Summary
If `minimumReleaseAge` setting is used, then pending version updates are paused until age criteria is satisfied:
- (https://github.com/renovatebot/renovate/discussions/30171)

Usually, it is possible to force PR creation by checking the checkbox in the "Dependency Dashboard" issue.

However, if PR is already open, but there are newer version that didn't pass the `minimumReleaseAge` , you can see them in the table in "Pending" column. In this case there is no way to force update to the latest one.

For example:

| Package | Update | Change | Pending |
|---|---|---|---|
| [loki](https://grafana.github.io/helm-charts) ([source](https://togithub.com/grafana/helm-charts)) | patch | `6.6.4` -> `6.6.5` | `6.7.1` (+2) |

Would be great to have a way to force upgrade to the latest one in this case.

### Proposal

PRs already have checkbox that allows you to rebase it:

>  - [ ] If you want to rebase/retry this PR, check this box

Having similar one that could force upgrade to the pending version, would solve the problem:

>  - [ ] If you want to update to the **Pending** version ignoring `minimumReleaseAge` setting, check this box

**Comment by HonkingGoose** [comment]
The maintainer wants to add a checkbox somewhere to force open pending PRs, see:

- #21719

It's not clear to me _where_ that checkbox will go, so I asked about it in the linked issue. :smile:

**Reply by mrclrchtr** [reply]
That's a feature I would very much like to have. 🚀
